File: 01_Language/01_Functions/python/is_a.py
# coding: utf-8

if __name__ == '__main__':
    class WidgetFactory(object):
        def __init__(self):
            self.oink = "moo"

    WF = WidgetFactory()
    if isinstance(WF, WidgetFactory):
        print("Yes")
    else:
        print("No")

File: 01_Language/01_Functions/php/is_a.php
<?php

// define a class
class WidgetFactory
{
    var $oink = 'moo';
}

// create a new object
$WF = new WidgetFactory();

if (is_a($WF, 'WidgetFactory')) {
    echo "yes, \$WF is still a WidgetFactory\n";
}
